x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/TransformIndex.java

```diff
@@ -281,7 +281,10 @@ private static Map<String, Object> createMetadata(String id, Clock clock) {
 
         Map<String, Object> transformMetadata = new HashMap<>();
         transformMetadata.put(TransformField.CREATION_DATE_MILLIS, clock.millis());
-        transformMetadata.put(TransformField.VERSION.getPreferredName(), Map.of(TransformField.CREATED, Version.CURRENT.toString()));
+        transformMetadata.put(
+            TransformField.VERSION.getPreferredName(),
+            Map.of(TransformField.CREATED, TransformConfigVersion.CURRENT.toString())
```

Comment:
Leave this file to https://github.com/elastic/elasticsearch/pull/97934

```suggestion
            Map.of(TransformField.CREATED, Version.CURRENT.toString())
```